libpromises/Makefile.am

```diff
@@ -129,7 +129,7 @@ libpromises_la_SOURCES = \
 	parser_state.h \
 	patches.c \
 	pipes.h pipes.c \
-	processes_select.c processes_select.h \
+	processes_select.h \
```

Comment:
As explained below, this removes the source from the windows build, you will need to add it back. Something like:

```
if NT
libpromises_la_SOURCES += processes_select_cmd.c
endif
```